api/proto/realtime/realtime.proto

```diff
@@ -293,6 +293,21 @@ message TimedOut {
 
 message DeclineMatchRequest { string request_id = 1; }
 
+// Tournaments
+
+enum TournamentGameResult {
+  // NO_RESULT: the game is not over
+  NO_RESULT = 0;
+  WIN = 1;
+  LOSS = 2;
+  DRAW = 3;
+  BYE = 4;
+  FORFEIT_WIN = 5;
+  FORFEIT_LOSS = 6;
+  // ELIMINATED: player is eliminated in a bracket tournament
+  ELIMINATED = 7;
+}
+
```

Comment:
we're likely going to want an actual end game result event here as well that can be sent to the frontend. We can even use GameEndedEvent although that one is a little heavyweight. lmk what you think.

Reply:
I'm adding this to `realtime.proto` in my mvp tourney branch. We'll coordinate soon to merge ours together; I'm trying to make as few changes as possible.

```proto
// TournamentGameEndedEvent gets sent to a tournament channel when a game is
// over. This shows tournament results right in the tournament lobby in real
// time.
message TournamentGameEndedEvent {
  string game_id = 1;
  map<string, int32> scores = 2;
  GameEndReason end_reason = 3;
  string winner = 4;
  string loser = 5;
  // If it was a tie, the winner and loser above can be interchanged.
  bool tie = 6;
  // Time that the game ended
  int64 time = 7;
}
```